Tasks/AzureCLIV2/azureclitask.ts

```diff
@@ -28,15 +29,35 @@ export class azureclitask {
             var connectedService: string = tl.getInput("connectedServiceNameARM", true);
             this.loginAzureRM(connectedService);
 
+            let errLinesCount: number = 0;
+            let aggregatedErrorLines: string[] = [];
+            tool.on('errline', (errorLine: string) => {
+                if (errLinesCount < 10) {
```

Comment:
Why're we limiting to 10 lines?

Reply:
yes this is limited to 10 similar to Powershell task to limit the number of error lines otherwise it will blot the entire Pipeline if the no. of lines are greater than that say 100 or so.

Reply:
Shouldn't we then show the last 10 lines? I think that'll be more useful

Reply:
Usually the top lines are useful in an error message. Bottom part is mostly stack trace.